tox.ini

```
@@ -0,0 +1,76 @@
+[tox]
+isolated_build = true
+envlist = py36, py37, py38, flake8, mypy, docs
+
+[testenv]
+extras =
```

I would propose to add a `description` keyword to describe this target. This is very helpful if you want to get a list of all available task and their description with `tox -av` as describe in one of my comments above.

```suggestion
description = Run the test suite
extras =
```

Same with the other targets. :wink:

---

Done :+1: